lang/syntax/src/trees/generic/lookup.rs

```diff
@@ -237,23 +238,23 @@ impl<'a, P: Phase> Item<'a, P> {
 }
 
 #[derive(Debug, Clone)]
-pub enum Type<'a, P: Phase> {
+pub enum Polarity<'a, P: Phase> {
```

Comment:
Note that this is not just the *polarity*, it is a union of the data/codata type definition tagged with the polarity.

Reply:
I think I had to rename it to something due to a name-clash with the universe `Type` in expressions. I will rename this to `DataCodata`, that might be a better name.